mlir/test/lib/Dialect/Test/TestAttributes.cpp

```diff
@@ -64,6 +66,25 @@ void CompoundAAttr::print(AsmPrinter &printer) const {
 // CompoundAAttr
 //===----------------------------------------------------------------------===//
 
+
+Attribute TestDecimalIntegerAttr::parse(AsmParser &parser, Type type) {
+  if (parser.parseLess()){
+    return Attribute();
+  }
+  uint64_t intVal;
+  if (failed(*parser.parseOptionalDecimalInteger(intVal))) {
+    return Attribute();
+  }
+  if (parser.parseGreater()) {
```

Comment:
Lets parse `x` and `a` (say) here too, that way you have a valid hex number and you verify that the reset worked as expected.

Reply:
Actually `5` instead of `a` means you can reuse test below but as a single positive one.

Reply:
I reworked it to look more like a shape like the dialect this is going to fix later